core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -71,6 +72,12 @@ private[spark] object CallSite {
 private[spark] object Utils extends Logging {
   val random = new Random()
 
+  // Use a common file numbering format which defaults to 5 digits for saving various part files.
+  val numFormatter = NumberFormat.getIntegerInstance()
+  numFormatter.setMinimumIntegerDigits(5)
+  numFormatter.setMaximumIntegerDigits(10)
```

Comment:
I don't think we really need to set the max.   Every place this is used we pass an id in that is already restricted to an int.  If we did happen to change those other interfaces to take say a long this would then break again.  I'm thinking we should just remove.